full_snapshot.proto

```diff
@@ -141,6 +143,23 @@ message DatabaseInformation {
   bool collected_local_catalog_data = 11;
 }
 
+message DatabaseStatistic {
+  int32 database_idx = 1;
+
+  uint32 transactions_per_sec = 2; // Transactions per second for this database
+  uint32 min_xid = 3; // Oldest transaction ID for this database
+  uint32 min_mxid = 4; // Oldest multixact ID for this database
+  uint32 xid_age = 5; // Oldest transaction age for this database
+  uint32 mxid_age = 6; // Oldest multixact age for this database
+}
+
+// Postgres server statistic
+// See System for server system statistic
+message ServerStatistic {
+  int64 current_xact_id = 1; // Current transaction ID for this server
+  int64 next_multi_xact_id = 2; // Next multixact ID for this server
```

Comment:
For both of these, it might be helpful if we clarify that they are the "full" transaction ID, i.e. the one that includes an epoch component and never wraps around (I believe its called `FullTransactionId` in the Postgres source)